autoload/lsp/omni.vim

```diff
@@ -332,141 +297,58 @@ function! lsp#omni#get_vim_completion_item(...) abort
     return call(g:lsp_get_vim_completion_item[0], a:000)
 endfunction
 
-augroup lsp_completion_item_text_edit
-    autocmd!
-    autocmd CompleteDone * call <SID>apply_text_edits()
-augroup END
-
-function! s:apply_text_edits() abort
-    " textEdit support function(callin from CompleteDone).
-    "
-    " expected user_data structure:
-    "     v:completed_item['user_data']: {
-    "       'vim-lsp/textEdit': {
-    "         'range': { ...(snip) },
-    "         'newText': 'yyy'
-    "       },
-    "       'vim-lsp/additionalTextEdits': [
-    "         {
-    "           'range': { ...(snip) },
-    "           'newText': 'yyy'
-    "         },
-    "         ...
-    "       ],
-    "     }
-    if !g:lsp_text_edit_enabled
-        doautocmd User lsp_complete_done
-        return
-    endif
+"
+" create item's user_data.
+"
+function! s:create_user_data(item, server_name) abort
+    let l:user_data = {}
 
-    " completion faild or not select complete item
-    if empty(v:completed_item)
-        doautocmd User lsp_complete_done
-        return
+    " InsertStartKey.
+    let l:user_data[s:user_data_insert_start_key] = -1
+    if has_key(a:item, 'textEdit') && type(a:item.textEdit) == type({})
```

Comment:
we use `['']` pattern through out the code base. can you change this to `a:item['textEdit']` to be consistent with the rest of the codebase?

Reply:
Thanks for your advice.
I will change the pattern for all occurrences.

Reply:
Fixed it.